### DISCUSSION TITLE
removal of setting osx_speed

### DISCUSSION THREAD

**Discussion by desepticon** [sabnzbd/sabnzbd#2214] [General]
I run SAbnzbd headless so have no need for this setting to be on. Just a waste of resources really. I imagine many people do the same.

**Comment by bulfinch** [comment]
I have no memory of making the change from default, but I don’t use a Mac so I may have been playing with the settings.

**Comment by Safihre** [comment]
@desepticon Actually, it doesn't work like that.. It still calculates the speed etc, but just doesn't show it. 
Would you not rather just disable the whole tray icon?

**Reply by desepticon** [reply]
I suppose that would be better.

**Reply by Safihre** [reply]
Will implement 1 setting for al operating systems for 3.7.0.

**Reply by Naden7** [reply]
When I first started my speed was 10mbs. I don't need this restriction anymore
